crt_portal/cts_forms/views.py

```diff
@@ -630,6 +633,34 @@ def post(self, request):
             return render(request, 'forms/complaint_view/actions/index.html', output)
 
 
+class SaveReportAttachmentView(LoginRequiredMixin, FormView):
```

Comment:
Opportunity here to specify, with something like [http_method_names](https://docs.djangoproject.com/en/2.2/ref/class-based-views/base/#django.views.generic.base.View.http_method_names), that we're only handling POST requests. So we return a 405 on anything else instead of a 500.